sonarts-sq-plugin/sonar-typescript-plugin/src/test/java/org/sonar/plugin/typescript/ContextualServerTest.java

```diff
@@ -19,13 +19,137 @@
  */
 package org.sonar.plugin.typescript;
 
+import java.io.File;
+import java.io.IOException;
+import org.junit.BeforeClass;
+import org.junit.Rule;
 import org.junit.Test;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.config.Configuration;
+import org.sonar.api.config.internal.MapSettings;
+import org.sonar.api.utils.internal.JUnitTempFolder;
+import org.sonar.api.utils.log.LogTester;
+import org.sonar.api.utils.log.LoggerLevel;
+import org.sonar.plugin.typescript.executable.ExecutableBundle;
+import org.sonar.plugin.typescript.executable.ExecutableBundleFactory;
+import org.sonar.plugin.typescript.executable.SonarTSCommand;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.sonar.plugin.typescript.TestUtils.resourceScript;
 
 public class ContextualServerTest {
```

Comment:
@saberduck let's add unit test for sensor as well

Reply:
let's postpone this